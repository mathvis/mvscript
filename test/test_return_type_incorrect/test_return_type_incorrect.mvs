func a() bool {
	return 26.0;
}
