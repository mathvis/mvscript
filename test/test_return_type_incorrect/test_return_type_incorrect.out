error: Expected bool but got float
 --> test/test_return_type_incorrect/test_return_type_incorrect.mvs:2:9
  |
2 | 	return 26.0;
  |     ^
  |
hint: Consider changing the return type or declaring a new function.
